db/views/searches_v09.sql

```diff
@@ -0,0 +1,47 @@
+WITH defendants AS (
+  SELECT
+    app.id,
+    CASE WHEN app.application_type = 'crm4' THEN
+        (app_ver.application -> 'defendant' ->> 'first_name') || ' ' || (app_ver.application -> 'defendant' ->> 'last_name')
+       ELSE
+        (
+          SELECT (defendants.value->>'first_name') || ' ' || (defendants.value->>'last_name')
+          FROM jsonb_array_elements(app_ver.application->'defendants') AS defendants
+          WHERE defendants.value->>'main' = 'true'
+        )
+       END AS client_name
+  FROM application AS app
+    JOIN application_version AS app_ver
+      ON app.id = app_ver.application_id AND app.current_version = app_ver.version
+)
+SELECT
+  app.id,
+  app_ver.id as application_version_id,
+  app_ver.application ->> 'ufn' as ufn,
+  app_ver.application ->> 'laa_reference' as laa_reference,
+  app_ver.application -> 'firm_office' ->> 'name' as firm_name,
+  app_ver.application -> 'firm_office' ->> 'account_number' as account_number,
+  app_ver.application ->> 'service_name' as service_name,
+  (app_ver.application -> 'cost_summary' ->> 'high_value')::boolean as high_value,
```

Comment:
There's a bit of double-casting here - you're taking `high_value`, which is a boolean, casting it to text with `->>`, and then casting it back to a boolean. Can we avoid this? (I suppose it might be that this is the only way to ensure the view doesn't explode if high_value turns out to have an unexpected type)